relayer-cli/src/components.rs

```diff
@@ -36,13 +37,14 @@ impl JsonTracing {
     #[allow(trivial_casts)]
     pub fn new(cfg: GlobalConfig) -> Result<Self, FrameworkError> {
         let filter = build_tracing_filter(cfg.log_level);
-        let use_color = false;
 
         // Construct a tracing subscriber with the supplied filter and enable reloading.
         let builder = FmtSubscriber::builder()
+            .with_target(false)
             .with_env_filter(filter)
             .with_writer(std::io::stderr as StdWriter)
-            .with_ansi(use_color)
+            // Note: JSON output is currently un-affected by ANSI 'color' option.
+            .with_ansi(COLORED_OUTPUT)
```

Comment:
```suggestion
            .with_ansi(false)
```

Reply:
Got it!

I made it false.

I also removed the `const` -- this no longer seems useful since we'll ignore it for the Json formatter. WDYT?

1299d8ff